addons/visibility/functions/fnc_start.sqf

```diff
@@ -0,0 +1,55 @@
+#include "script_component.hpp"
+/*
+ * Author: Brett
+ * Starts the Draw3D eh that indicates when a player can see the curator cursor
+ *
+ * Example:
+ * [] call zen_visibility_fnc_start
+ *
+ * Public: No
+ */
+
+if (GVAR(draw) != -1) exitWith {};
+
+GVAR(draw) = addMissionEventHandler ["Draw3D", {
```

Comment:
This needs to be put into a separate function. [reference](https://ace3mod.com/wiki/development/coding-guidelines.html#73-event-driven)